src/pybind/mgr/cephadm/module.py

```diff
@@ -1266,6 +1269,11 @@ def _run_cephadm(self, host, entity, command, args,
             self.log.debug('%s container image %s' % (entity, image))
 
             final_args = []
+
+            if env_vars:
+                for env_var_pair in env_vars:
+                    final_args.extend(['--env', env_var_pair])
```

Comment:
```suggestion
            if env_vars:
                for k,v in env_vars.items():
                    final_args.extend(['--env', f'{k}={v}'])
```

Maybe it would be better to use a dict type instead of list?

Reply:
I deliberately chose [`K=V`] as this is the format it will be consumed and passed along by bin/cephadm. While I do agree that a dict type is the correct type for this case, but in this case I'd favor practicability.